docs/installation.rst

```diff
@@ -35,9 +41,10 @@ For security reasons, many HPCs (e.g., TACC) do not allow Docker containers, but
 In this case, start with a machine (e.g., your personal computer) with Docker installed.
 Use `docker2singularity <https://github.com/singularityware/docker2singularity>`_ to create a singularity image. You will need an active internet connection and some time. ::
 
-    $ docker run -v /var/run/docker.sock:/var/run/docker.sock \
-        -v D:\host\path\where\to\ouptut\singularity\image:/output \
-        --privileged -t --rm singularityware/docker2singularity \
+    $ docker run --privileged -t --rm \
+        -v /var/run/docker.sock:/var/run/docker.sock \
+        -v D:\host\path\where\to\output\singularity\image:/output \
```

Comment:
Does docker2singularity use a `D:\`-style filesystem for the host, or is this assuming a Windows user?